tracpro/polls/tests/test_views.py

```diff
@@ -127,7 +145,8 @@ def test_fetching_pollruns_csv(self):
         self.assertEqual(200, response.status_code)
         self.assertEqual('text/csv', response['Content-Type'])
 
-        rows = [row for row in csv.reader(StringIO(response.content.decode('utf-8')))]
+        rows = [[element.decode('utf-8') for element in row]
+                for row in csv.reader(StringIO(response.content.decode('utf-8')))]
```

Comment:
`row[0]` (line 150, below) would also contain 'Cohorts' instead of 'Cohort' when changed in views.